### ISSUE TITLE
Error: No valid exports main found for '/usr/local/lib/node_modules/aws-cdk/node_modules/uuid'

### ISSUE THREAD

**Issue by nirsnappy** [aws/aws-cdk#8300]
any usage return error : Error: No valid exports main found for '/usr/local/lib/node_modules/aws-cdk/node_modules/uuid'

### Reproduction Steps

```
npm i -g aws-cdk
cdk
```

### Error Log

 Error: No valid exports main found for '/usr/local/lib/node_modules/aws-cdk/node_modules/uuid'

### Environment

  - **CLI Version      :1.42.0
  - **Framework Version:1.42.0
  - **OS               :macos 10.15.3
  - **Language         :**

### Other

```
➜  tests cdk
Error: No valid exports main found for '/usr/local/lib/node_modules/aws-cdk/node_modules/uuid'
    at resolveExportsTarget (internal/modules/cjs/loader.js:625:9)
    at applyExports (internal/modules/cjs/loader.js:502:14)
    at resolveExports (internal/modules/cjs/loader.js:551:12)
    at Function.Module._findPath (internal/modules/cjs/loader.js:657:22)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:960:27)
    at Function.Module._load (internal/modules/cjs/loader.js:855:27)
    at Module.require (internal/modules/cjs/loader.js:1033:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk/lib/api/deploy-stack.ts:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1144:30)
```

--- 

This is :bug: Bug Report

**Comment by shivlaks** [comment]
@nirsnappy what version of Node.js are you using?

**Comment by niradler** [comment]
Reinstall node fix the issue, node version was around 14.1 sorry its too late now to check this.

**Comment by shivlaks** [comment]
tested on `14.1.0` which we say we support and did not run into this issue when installing.

I'm resolving as a duplicate of #7816 as signs point to previous Node version being (13.x).
Node `13.x` has reached EOL as of 5/30/2020

Reopen as needed.
